client/src/utils/apiWrapper.js

```diff
@@ -94,3 +94,16 @@ export const getMemberSchemaTypes = () => {
       error,
     }));
 };
+
+// Retrieves a member's permissions from their mongo ID
+export const updateMember = (member, memberID) => {
+  const requestString = `${BACKEND_BASE_URL}/members/${memberID}`;
+  return axios
+    .put(requestString, {
+      ...member,
```

Comment:
small thing, but instead of `{...member}` you could just do `member`

Reply:
Actually, it has to be ```...member```. The endpoint is expecting the request to be the member object, however, without using the spread operator, the request contains the key "member" pointing to the member object, which causes a server error